service/src/main/java/bio/terra/landingzone/stairway/flight/create/resource/step/AggregateLandingZoneResourcesStep.java

```diff
@@ -0,0 +1,58 @@
+package bio.terra.landingzone.stairway.flight.create.resource.step;
+
+import bio.terra.landingzone.job.JobMapKeys;
+import bio.terra.landingzone.service.landingzone.azure.model.DeployedLandingZone;
+import bio.terra.landingzone.service.landingzone.azure.model.LandingZoneResource;
+import bio.terra.landingzone.stairway.flight.LandingZoneFlightMapKeys;
+import bio.terra.stairway.FlightContext;
+import bio.terra.stairway.Step;
+import bio.terra.stairway.StepResult;
+import bio.terra.stairway.exception.RetryException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+
+public class AggregateLandingZoneResourcesStep implements Step {
```

Comment:
Why do we have this step vs. individual steps storing their own resources?

Reply:
You are right each step already save its own result. We need to combine all those saved resource info into one result. I think this is convenient way of doing it. Also it allows to make data transformation if needed. And decided what resources should be shown as a result. For instance, show only shared resources.